pydantic/type_adapter.py

```diff
@@ -23,6 +39,7 @@
 from .plugin._schema_validator import create_schema_validator
 
 T = TypeVar('T')
+R = TypeVar('R')
```

Comment:
Why `R`, out of curiosity? Is this convention for the 2nd type var? I'd assume we'd use `S` or something adjacent to `T`, haha.

Reply:
Took `R` as in Return type. This kind of `R` is also used in `plugin/_schema_validation.py` for a Callable return type